hs_core/templates/pages/create-resource.html

```diff
@@ -48,14 +48,14 @@
         <div class="col-sm-10">
             <select class="form-control" name="resource-type" id="resource-type">
                 <option value="GenericResource">Generic</option>
-                <option value="RasterResource">Geographic Raster</option>
-                <option value="RefTimeSeries">HIS Referenced Time Series</option>
-                <option value="ModelProgramResource">Model Program</option>
-                <option value="ModelInstanceResource">Model Instance</option>
-                <option value="SWATModelInstanceResource">SWAT Model Instance</option>
-                <option value="NetcdfResource">Multidimensional (NetCDF)</option>
-                <option value="TimeSeriesResource">Time Series</option>
-                <option value="ToolResource">Application</option>
+{#                <option value="RasterResource">Geographic Raster</option>#}
+{#                <option value="RefTimeSeries">HIS Referenced Time Series</option>#}
+{#                <option value="ModelProgramResource">Model Program</option>#}
+{#                <option value="ModelInstanceResource">Model Instance</option>#}
+{#                <option value="SWATModelInstanceResource">SWAT Model Instance</option>#}
+{#                <option value="NetcdfResource">Multidimensional (NetCDF)</option>#}
+{#                <option value="TimeSeriesResource">Time Series</option>#}
+{#                <option value="ToolResource">Application</option>#}

```

Comment:
Are are these being hidden right now? Is this to disable under they are merged into the concrete tree?